scripts/generate_black_spots_input.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python
+
+"""Loads records and roads shapefile, outputs data needed for generating black spots"""
+import argparse
+import csv
+from dateutil import parser
+from dateutil.relativedelta import relativedelta
+import fiona
```

Comment:
We haven't done this with past import scripts on this project but this one has enough dependencies that I wonder if it might be a good idea to install them into the `app` or `celery` VM.